include/zephyr/posix/netinet/in.h

```diff
@@ -17,6 +17,29 @@ extern "C" {
 typedef uint16_t in_port_t;
 typedef uint32_t in_addr_t;
 
+#define IN6_IS_ADDR_UNSPECIFIED(addr) \
+(((const struct in6_addr *)addr)->s6_addr32[0] == 0           \
+&& ((const struct in6_addr *)addr)->s6_addr32[1] == 0         \
+&& ((const struct in6_addr *)addr)->s6_addr32[2] == 0         \
+&& ((const struct in6_addr *)addr)->s6_addr32[3] == 0)
+
+#define IN6_IS_ADDR_LOOPBACK(addr) \
+(((const struct in6_addr *)addr)->s6_addr32[0] == 0           \
+&& ((const struct in6_addr *)addr)->s6_addr32[1] == 0         \
+&& ((const struct in6_addr *)addr)->s6_addr32[2] == 0         \
+&& ((const struct in6_addr *)addr)->s6_addr32[3] == htonl(1))
+
+#define IN6_IS_ADDR_LINKLOCAL(addr) \
+(((const struct in6_addr *)addr)->s6_addr32[0] & htonl(0xffc) == htonl(0xfe8))
+
+#define IN6_IS_ADDR_SITELOCAL(addr) \
+(((const struct in6_addr *)addr)->s6_addr32[0] & htonl(0xffc) == htonl(0xfec))
+
+#define IN6_IS_ADDR_V4MAPPED(addr) \
```

Comment:
I recently added these to `include/zephyr/net/socket.h` by commit 8f49925a0fcc

Reply:
Oh, I missed that, thanks. Removed it from this PR.